### DISCUSSION TITLE
Welcome to Feel++ Discussions!

### DISCUSSION THREAD

**Discussion by prudhomm** [feelpp/feelpp#1547] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by romainhild** [comment]
Is that supposed to replace the support thread on slack ?

**Reply by prudhomm** [reply]
No. This is a forum for discussions and Q&A. Slack remains our chat tool for our projects. Typically, Users or Wannabee Users would use it to ask questions. You can look for questions and answers whereas in slack we have a limited amount of threads and answers disappear in time. Here we have a nice archive or Q&A plus discussions on various aspects of feelpp and possible feedback  on our work. 

- There is also a category for showing off results @romainhild. 
- We can make links to issues or commits.
- We can reference discussion in corresponding issues

**Reply by romainhild** [reply]
ok, but it looks a lot like the #support channel of slack then, except that all the answers will stay.
For the discussions on the different projects, Slack is fine to chat, but I think that for the support, it is good that the answers stay, so that we don't have to repeat questions after a while.

**Reply by prudhomm** [reply]
exactly 👍🏻

**Reply by prudhomm** [reply]
#support allows to ask a question in a chat and depending on subject it may be indeed interesting to move it to Discussions!
